docs/_static/vcr/v1.yaml

```diff
@@ -220,3 +226,27 @@ components:
         proof:
           description: one or multiple cryptographic proofs
           type: object
+    IssueVCRequest:
+      type: object
+      description: A request for issuing a new Verifiable Credential.
+      required:
+        - type
+        - issuer
+        - credentialSubject
+      properties:
+        type:
```

Comment:
must be single type, not a list. According to spec we only accept one additional type.